### PR TITLE
RELATED: RAIL-1289 Breaking UI of pivot table when scroll horizontal bar quickly after clicked aggregate on header menu

### PR DESCRIPTION
**Pull request by Hurtak**

### CODE REVIEW

**Comment by Hurtak** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2911/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2284/ : SUCCESS in 4m 23s

File: src/components/core/PivotTable.tsx [hunk 1]
```diff
@@ -382,6 +382,27 @@ export const getDrillIntersection = (
     });
 };
 
+function measureCoulumnReadyToRender(params: any, execution: Execution.IExecutionResponses): boolean {
```
Comment (no23reason) [inline src/components/core/PivotTable.tsx]: Nitpick: maybe `isMeasureCoulumnReadyToRender` ?

**Comment by no23reason** [comment]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

File: src/components/core/PivotTable.tsx (hunk 1 above)
Comment (Hurtak) [reply src/components/core/PivotTable.tsx]: Renamed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2914/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2285/ : SUCCESS in 2m 13s

**Comment by Hurtak** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1202/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2915/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/711/ : SUCCESS in 19m 58s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2916/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2286/ : SUCCESS in 2m 57s

**Comment by Hurtak** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1203/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2918/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/712/ : SUCCESS in 19m 25s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/627/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2288/ : SUCCESS in 11m 54s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1204/ : SUCCESS in 29m 01s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/600/ : SUCCESS in 27m 13s

**Pull request merged**